.github/workflows/zxc-compile-code.yaml

```diff
@@ -64,13 +64,13 @@ permissions:
 
 jobs:
   compile:
-    name: ${{ inputs.custom-job-label || 'Compiles' }}
-    runs-on: ${{ matrix.os }}
     strategy:
       matrix:
         os:
           - windows-2022
           - [self-hosted, Linux, large, ephemeral]
+    runs-on: ${{ matrix.os }}
+    name: "(${{ join(matrix.os, ',') }})"
```

Comment:
Nathan gave me the feedback to have name first, then runs-on

Reply:
Updated